extensions/amp-sidebar/1.0/component.js

```diff
@@ -166,11 +167,81 @@ function SidebarWithRef(
         >
           <div className={classes.backdropOverscrollBlocker}></div>
         </div>
-      </>
-    )
+      </div>
+    </>
   );
 }
 
 const Sidebar = forwardRef(SidebarWithRef);
 Sidebar.displayName = 'Sidebar'; // Make findable for tests.
 export {Sidebar};
+
+/**
+ * @param {!SidebarDef.SidebarNavToolbar} props
+ * @return {PreactDef.Renderable}
+ */
+export function SidebarNavToolbar({
+  toolbar,
+  'toolbar-target': toolbarTarget,
+  children,
+  ...rest
+}) {
+  const ref = useRef();
+  const [targetEl, setTargetEl] = useState();
+  const [, setReRender] = useState(true);
```

Comment:
Use this state variable to trigger re-render.  It toggles between `true` and `false`, but mainly just need to update its value to cause a re-render.